util/missingcrypto111.txt

```diff
@@ -1213,7 +1213,7 @@ TS_TST_INFO_set_serial
 TS_TST_INFO_set_time
 TS_TST_INFO_set_tsa
 TS_TST_INFO_set_version
-TS_VERIFY_CTS_set_certs
+TS_VERIFY_CTX_set_certs
```

Comment:
The remaining travis failures are doc-nits errors, because now the macro `TS_VERIFY_CTS_set_certs()` is undocumented.

```
include/openssl/ts.h: macro TS_VERIFY_CTS_set_certs undocumented
# 1 macros undocumented (count is approximate)
make: *** [doc-nits] Error 1
```

In view of the fact that @InfoHunter already opened #9838, I would suggest that @mattcaswell makes an exception from https://github.com/openssl/openssl/pull/9472#discussion_r319386886 and you can just leave both names in here for the moment @Iniesta8. I will drop a reminder on #9838 that the compatibility macro needs to be documented, including the deprecation.

Reply:
No problem, I will update the document and I think #9838 should be merged after this PR is merged.

Reply:
The errors still occur...